src/pages/jobLogs/PaginationButtons.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import { Disclaimer } from "@leafygreen-ui/typography";
+import Icon from "components/Icon";
+import { size } from "constants/tokens";
+import { useUpdateURLQueryParams } from "hooks/useUpdateURLQueryParams";
+
+interface PaginationButtonsProps {
+  numPages: number;
+  currentPage: number;
+}
+
+export const PaginationButtons: React.FC<PaginationButtonsProps> = ({
```

Comment:
This is becoming increasingly common across a bunch of components we should eventually turn this into a reusable component.